README.md

```diff
@@ -9,7 +9,7 @@
 # 📗 Table of Contents
 
 - [📗 Table of Contents](#-table-of-contents)
-- [🎥 RSL Media ](#-rsl-media-)
+- [ 💻 Math Magicians ](#-math-magicians-)
```

Comment:
- [ ] Kindly remove the following highlighted boilerplate files as they would not be needed.

  ![image](https://user-images.githubusercontent.com/36542986/211525159-a72fe099-347d-4d27-803d-fa4f649b2fc7.png)